requirements.txt

```diff
@@ -1,20 +1,46 @@
-# Application
-awscli
-docopt
-edx-ccx-keys==1.1.0 # Pinned for python 2.7 compatibility
-edx-opaque-keys==2.1.1 # Pinned for python 2.7 compatibility
-graphitesend==0.10.0 # Apache
-path.py
-pbr
-pip==1.5.6		# MIT // AN-4322
-pymongo
-python-dateutil
-python-gnupg==0.3.6
-pyyaml
-psutil
-
-# Tests
-coverage==4.3.4
-mock==2.0.0
-pytest==3.0.7
-pytest-cov==2.4.0
+## Original Requirements
+# awscli
+# docopt
+# edx-ccx-keys==1.1.0 # Pinned for python 2.7 compatibility
+# edx-opaque-keys==2.1.1 # Pinned for python 2.7 compatibility
+# graphitesend==0.10.0 # Apache
+# path.py
+# pbr
+# pip==1.5.6		# MIT // AN-4322
+# pymongo
+# python-dateutil
+# python-gnupg==0.3.6
+# pyyaml
+# psutil
+
+# # Tests
+# coverage==4.3.4
+# mock==2.0.0
+# pytest==3.0.7
+# pytest-cov==2.4.0
+
+
+
+## Modified Requirements satisfied by edx-platform
+# boto3==1.4.8
+# docopt==0.6.2
+# edx-ccx-keys==1.2.0 # Pinned for python 2.7 compatibility
+# edx-opaque-keys==2.2.1 # Pinned for python 2.7 compatibility
+# path.py==12.5.0
+# pbr==5.5.1
+# pymongo==3.10.1
+# python-dateutil==2.4.0
+# pyyaml==5.4.1
+# psutil==5.8.0
+
+# # Tests
+# coverage==5.5
+# mock==4.0.3
+# pytest==6.2.3
+# pytest-cov==2.11.1
```

Comment:
Could you please add a brief comment about why you've commented this out?

Reply:
Done